### ISSUE TITLE
Create more adjustments

### ISSUE THREAD

**Issue by JennyMeissner** [CommandPost/LoupedeckPlugin#20]
Create adjustments that currently exist only through press actions:

- [x] Change Marker (= Next / Previous Marker)
- [x] Nudge Marker
- [x] Nudge (= Nudge Left / Right in Trim group)
- [x] Nudge Vertically (= Nudge Up/Down)
- [x] Change Edit (=Next / Previous Edit)
- [x] Change Bank
- [x] Change Keyframe
- [x] Change Focus Point
- [x] Change Field
- [x] Playhead: Subframe
- [x] Change Clip
- [x] Pan
- [x] Tilt
- [x] Slide
- [x] Select Edge (?)
- [x] Select Video Edge (?)
- [x] Select Audio Edge (?)
- [x] Adjust Waveform Height
- [x] Move 10 frames
- [ ] Timeline History
- [ ] Roll

Basically, create adjustment from press actions that include term "left / right", "next / previous", "up / down"

**Comment by JennyMeissner** [comment]
- [x] "Move 10 frames in timeline" I cannot find either.

**Comment by JennyMeissner** [comment]
- [x] "Adjust Waveform Height" could be added too.

**Comment by latenitefilms** [comment]
These will be included in the next beta.

**Comment by JennyMeissner** [comment]
I added Roll and Timeline History 😬
